test/fixtures/cli-plugin-array/.gitattributes

```diff
@@ -0,0 +1 @@
+*.html text=auto eol=lf
```

Comment:
Even on windows the output seems to contain just `\n`. The problem is that if the line ending is setting to autocrlf the baseline files will have `\r\n` so the tests fails. This change forces `\n` line endings for html files in the subtree fixing the line ending mismatch.